Marlin/Marlin_main.cpp

```diff
@@ -5053,13 +5053,15 @@ inline void gcode_M226() {
 inline void gcode_M303() {
   int e = code_seen('E') ? code_value_short() : 0;
   int c = code_seen('C') ? code_value_short() : 5;
+  bool u = code_seen('U') && code_value_short() == 1;
```

Comment:
I _suppose_ `code_value_short() != 0` is fine here, then any non-zero value means "yes." Though technically, it should be "1".